keyboards/argo_works/ishi/80/mk0_rp2040_extra/info.json

```diff
@@ -0,0 +1,116 @@
+ {
+    "manufacturer": "Argo Works",
+    "keyboard_name": "Ishi 80 Mk 0",
+    "maintainer": "jasonj2232",
+    "development_board": "elite_c",
```

Comment:
Is this not intended to be setup for an RP2040?
This directory is identical to `mk0_avr_extra`...
```diff
diff mk0_rp2040_extra/info.json mk0_avr_extra/info.json
5a6
>     "pin_compatible": "elite_c",
Common subdirectories: mk0_rp2040_extra/keymaps and mk0_avr_extra/keymaps
diff mk0_rp2040_extra/rules.mk mk0_avr_extra/rules.mk
3c3,4
< WPM_ENABLE = yes
\ No newline at end of file
---
> WPM_ENABLE = yes
> PIN_COMPATIBLE = elite_c
\ No newline at end of file
```

Reply:
Ahh this directory wasn't meant to be included in the commit. I was using it for some testing, specifically with OLED stuff. I'll move it to another branch and remove it from the current one.